keyboards/westfoxtrot/cypher/rev1/readme.md

```diff
@@ -0,0 +1,15 @@
+# cypher
+
+![cypher](https://github.com/westfoxtrot/Cypher_PCB)
```

Comment:
This needs to be an actual image.

Reply:
What do you mean an image? A picture of the PCB? Can I just not have a link instead

Reply:
The `![]()` syntax is for an image. If you don't have an image to put here, just remove this line, as the link is already here in the hardware availability section.